### ISSUE TITLE
api_helper_functions.lua:128: attempt to index local 'tiles' (a nil value)

### ISSUE THREAD

**Issue by Bastrabun** [runsy/petz#140]
Can you please test before release? :P

```
2023-01-16 09:16:45: ERROR[Main]: ServerError: AsyncErr: Lua: Runtime error from mod '' in callback luaentity_run_simple_callback(): ..._test/bin/../mods/petz/kitz/api/api_helper_functions.lua:128: attempt to index local 'tiles' (a nil value)
2023-01-16 09:16:45: ERROR[Main]: stack traceback:
2023-01-16 09:16:45: ERROR[Main]: ..._test/bin/../mods/petz/kitz/api/api_helper_functions.lua:128: in function 'create_inv_cube'
2023-01-16 09:16:45: ERROR[Main]: ....1/Minetest_test/bin/../mods/petz/petz/api/api_forms.lua:187: in function 'create_form'
2023-01-16 09:16:45: ERROR[Main]: ...est_test/bin/../mods/petz/petz/api/api_on_rightclick.lua:130: in function 'on_rightclick'
2023-01-16 09:16:45: ERROR[Main]: ...inetest_test/bin/../mods/petz/petz/petz/camel_mobkit.lua:105: in function 'func'
2023-01-16 09:16:45: ERROR[Main]: ...inetest_test/bin/../builtin/profiler/instrumentation.lua:107: in function <...inetest_test/bin/../builtin/profiler/instrumentation.lua:100>
```

Fore my reference 3534

**Comment by blaboing** [comment]
happened while testing fix for #138 
dirt/cobblestone as price worked fine, set price to stick after that.
crashed at "buyer" openining the shop formspec

**Comment by runsy** [comment]
Fixed.
